uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -458,20 +482,92 @@
         });
       });
 
+      describe('when manager opens leave request popup', function () {
+        beforeEach(function () {
+          //waiting approval request at index 3 with value 3
+          var leaveRequest = LeaveRequestInstance.init(mockData.all().values[3]);
```

Comment:
If we change order of mockData test case needs to be rewritten. instead write a find function to get the intended object.